tests/test_10_serializers.py

```diff
@@ -30,12 +31,23 @@ def fake_get_active_message(*args, **kwargs) -> None:
     )
 
 
+def fake_process_sanity_check(*args, **kwargs) -> dict[str, str | None]:
```

Comment:
@alfredotoselli this is not a stac_fastapi serializer test, move it to a new `test_sanity_check` file